CMSgov/cms-carts-seds
# MDCT-99: a11y Testing

# Description

Closes out [MDCT-99]([MDCT-99](https://qmacbis.atlassian.net/browse/MDCT-99)). In this PR, the following accessibility tools are installed:

- `axe-core` 
- `jest-axe`
- `pa11y`
- `HTML CodeSniffer`

The README is also updated to reflect changes and provide documentation. 

## How to test

`cd services/ui-src`
`nvm use`
`yarn install`
`yarn pa11y <url>`

Provide any endpoint in the command line to see a `pa11y` accessibility report. The other libraries should be listed in the `package.json` (no unit tests are available as of this moment).

## Dependencies 

Run `yarn install` 

# Get it done

We get to write aXe unit tests!

## Code authors checklist

## Reviewers Checklist (Two different people)

## Assignee
